client/src/components/CoursePane/CourseRenderPane.js

```diff
@@ -182,9 +184,7 @@ class CourseRenderPane extends PureComponent {
         if (this.state.loading) {
             currentView = (
                 <div className={classes.loadingGifStyle}>
-                    <video autoPlay loop>
-                        <source src={loadingGif} type="video/mp4" />
-                    </video>
+                    <img src={AppStore.getDarkMode() ? darkModeLoadingGif : loadingGif} alt="Loading courses" />
```

Comment:
Is there a better way to determine if the site is in dark mode?

Reply:
I don't see a problem with this approach?